app/src/main/java/de/xikolo/models/Video.kt

```diff
@@ -0,0 +1,131 @@
+package de.xikolo.models
+
+import com.squareup.moshi.Json
+import de.xikolo.models.base.RealmAdapter
+import de.xikolo.models.dao.ItemDao.Unmanaged.Companion.findForContent
+import de.xikolo.utils.LanguageUtil
+import io.realm.RealmList
+import io.realm.RealmObject
+import io.realm.annotations.Ignore
+import io.realm.annotations.PrimaryKey
+import moe.banana.jsonapi2.JsonApi
+import moe.banana.jsonapi2.Resource
+
+open class Video : RealmObject() {
+
+    @PrimaryKey
+    var id: String? = null
+
+    var summary: String? = null
+
+    var duration: Int = 0
+
+    var singleStream: VideoStream? = null
+
+    var lecturerStream: VideoStream? = null
+
+    var slidesStream: VideoStream? = null
+
+    var slidesUrl: String? = null
+
+    var slidesSize: Int = 0
+
+    var audioUrl: String? = null
+
+    var audioSize: Int = 0
+
+    var transcriptUrl: String? = null
+
+    var transcriptSize: Int = 0
+
+    var thumbnailUrl: String? = null
+
+    var subtitles = RealmList<VideoSubtitles>()
+
+    @Ignore
+    var item: Item? = findForContent(id)
```

Comment:
Should have a getter. This way the field is initialized on class init where `id` might be `null` and any call to `item` always returns `null`.